src/app/scripts/cac/control/cac-control-directions.js

```diff
@@ -386,6 +405,25 @@ CAC.Control.SidebarDirections = (function (_, $, Control, BikeModeOptions, Geoco
     }
 
     function onTypeaheadSelected(event, key, location) {
+
+        event.preventDefault();  // do not submit form
+
+        var $input;
+        var prefKey;
+
+        // Make sure to keep the directionsFrom origin in sync with the explore origin
+        if (key === 'origin') {
+            prefKey = 'origin';
+            $input = $(options.selectors.directionsFrom);
+        } else if (key === 'destination') {
+            prefKey = 'destination';
+            $input = $(options.selectors.typeaheadTo);
+        } else {
+            console.error('unrecognized typeahead key ' + key);
+            return;
+        }
```

Comment:
What is this for?  Since neither `$input` nor `prefKey` are used below, it seems like it's not doing anything except validating that the key is one of `origin` or `destination`.